cypress/e2e/data-export/display-list-of-settings-pages-permissions.cy.js

```diff
@@ -0,0 +1,50 @@
+import permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import devTeams from '../../support/dictionary/devTeams';
+import testTypes from '../../support/dictionary/testTypes';
+import dataExportPermission from '../../support/fragments/data-export/data-export-permission';
```

Comment:
when importing fragments, please do so with upper case naming style, for example: DataExportPermission

Reply:
Fixed.